data/sql/updates/pending_db_world/rev_1618383191741891235.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618383191741891235');
+
+UPDATE `quest_template_addon` SET `AllowableClasses` = 1090 WHERE (`ID` = 8989);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 9 WHERE (`ID` = 8990);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 144 WHERE (`ID` = 8991);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 260 WHERE (`ID` = 8992);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 9 WHERE (`ID` = 8985);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 1090 WHERE (`ID` = 8986);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 144 WHERE (`ID` = 8987);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 260 WHERE (`ID` = 8988);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 129 WHERE (`ID` = 8966);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 1280 WHERE (`ID` = 8967);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 12 WHERE (`ID` = 8968);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 82 WHERE (`ID` = 8969);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 129 WHERE (`ID` = 8962);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 1280 WHERE (`ID` = 8963);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 12 WHERE (`ID` = 8964);
+UPDATE `quest_template_addon` SET `AllowableClasses` = 82 WHERE (`ID` = 8965);
```

Comment:
@betayi you can group the same queries using, for example:

```sql
UPDATE `quest_template_addon` SET `AllowableClasses` = 9 WHERE `ID` IN (8990, 8985);
```

Reply:
thanks，cause i collect these sql with keira  quest editor ,copy and paste the sql one by one  .
and the value is a bit mask not like each other , so it's not necessary to group them .